box-ui/src/components/ts/Student/StudentMeeting.tsx

```diff
@@ -26,16 +28,17 @@ const StudentMeeting: FunctionComponent = () => {
     });
 
     const [selectCoord, setSelectCoord] = useState<boolean>(false);
-    const [coord, setCoord] = useState<Array<Array<number>>>([]);
+    const [coord, setCoord] = useState<[number, number][]>([]);
 
     //circle : svg element to display on click on the image
     const [circles, setCircles] = useState<React.SVGProps<SVGCircleElement>[]>([]);
-    const [endCarre, setEndCarre] = useState<boolean>(false);
     const [textBtn, setTextBtn] = useState<string>('Recadrer');
 
     // img : downloaded from back, potentially cropped
     const [height_img, setHeight] = useState<number>(0);
```

Comment:
You should use camelCase. The setter should also have the same name as the getter.
```suggestion
    const [heightImg, setHeightImg] = useState<number>(0);
```